app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Currently, your application does not ask for confirmation of email during signup. Kindly check out this [blog](https://web-crunch.com/posts/extending-devise-confirmation-emails) on how to set up devise confirmable so that your application asks for confirmation of `email`. Do not forget to add `:confirmable` here too.

Reply:
This issue is well resolved! ✔️